package.json

```diff
@@ -55,12 +55,15 @@
 	},
 	"dependencies": {
 		"@floating-ui/core": "^1.2.6",
-		"@floating-ui/dom": "^1.2.6"
+		"@floating-ui/dom": "^1.2.6",
+		"@fontsource/overpass": "^4.5.11",
+		"@fontsource/overpass-mono": "^4.5.11",
+		"@fontsource/source-code-pro": "^4.5.14",
+		"@iconify-json/lucide": "^1.1.82",
+		"@iconify-json/simple-icons": "^1.1.50"
```

Comment:
These should be devDependencies, since the `radix-svelte` package doesn't depend on them